api/resources/translations/messages-ne.properties

```diff
@@ -18,6 +18,13 @@ Analytics = विश्लेषण
 Antenatal\ Care = पुर्व प्रसूति सेवा
 Any\ date = मिति
 Any\ status = कुनै अवस्था
+api.startup.checks = इन्स्टलेसन जाँच
+api.startup.config = कन्फिगरेसन राख्दै
+api.startup.forms = फर्म हरु इन्स्टल गर्दै
+api.startup.install = इन्स्टल गर्दै
+api.startup.index = डाटा Indexing गर्दै
```

Comment:
Indexing to अनुक्रमणिका? I was searching myself as I did not know and seems this might be it but not sure.

Reply:
I tried to make it more meaningful and readable than actual google translate, which is harder for even ourselves to understand. May be, `डाटा इन्डेक्स गर्दै` ?

Reply:
Yes, let's avoid complex terminologies.

Reply:
I agree, डाटा इन्डेक्स गर्दै sounds good @1yuv